frontend/android/src/main/java/com/jakewharton/sdksearch/ui/MainActivity.kt

```diff
@@ -192,6 +197,15 @@ class MainActivity : Activity() {
     disposables.clear()
   }
 
+  override fun onConfigurationChanged(newConfig: Configuration?) {
```

Comment:
I think it is safe to make `newConfig: Configuration?` be `newConfig: Configuration` and remove the `!!.` or please use `?.`.